WcaOnRails/db/migrate/20161117085757_cleanup_events_issue96.rb

```diff
@@ -0,0 +1,12 @@
+class CleanupEventsIssue96 < ActiveRecord::Migration
+  def up
+    remove_foreign_key :preferred_formats, column: :event_id
+    remove_foreign_key :competition_events, column: :event_id
```

Comment:
I expect this to be controversial. Let's have a discussion.

I see this as a compelling reason to *not* have foreign keys in our database. It makes this migration impossible to write in the way I did. For those of you who are fans of foreign keys... how would you write this migration? @pingskills @FatBoyXPC.